.github/workflows/deploy.yml

```diff
@@ -119,6 +119,12 @@ jobs:
           kurtosis run --enclave cdk-v1 --args-file params.yml .
           yq -Y --in-place '.deploy_zkevm_permissionless_node = false' params.yml # reset
 
+      - name: Deploy CDK erigon node
+        run: |
+          yq -Y --in-place '.deploy_cdk_erigon_node = true' params.yml
+          kurtosis run --enclave cdk-v1 --args-file params.yml .
+          yq -Y --in-place '.deploy_cdk_erigon_node = false' params.yml # reset
```

Comment:
curious, why are we forced to reset like this?

Reply:
It's the incremental CI job, it deploys each step one by one